### PR TITLE
implemented setting custom color for rating stars

### PR DESCRIPTION
**Pull request by pglebocki**
Make sure these boxes are checked before submitting your PR - thank you!

- [ ] Pull request is being merged to the `develop` branch.
- [ ] There's a reference to an *Issue* which this solves.
- [ ] For new features a sample has been added to the sample app
illustrating this feature's behaviour.
- [ ] Commits are squashed to a single commit.

### CODE REVIEW

File: app-rating/src/main/java/com/stepstone/apprating/AppRatingDialogView.kt [hunk 1]
```diff
@@ -211,15 +197,23 @@ class AppRatingDialogView(context: Context) : LinearLayout(context), OnRatingBar
...
+        noteDescriptionText = findViewById(R.id.app_rate_dialog_note_description)
         ratingBar.setIsIndicator(false)
         ratingBar.setOnRatingBarChangeListener(this)
     }
 
     private val theme: Resources.Theme
         get() = context.theme
+
+    private fun getColorFromRes(@ColorRes colorResId: Int): Int {
+        return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```
Comment (zawadz88) [inline app-rating/src/main/java/com/stepstone/apprating/AppRatingDialogView.kt]: You could use `ResourcesCompat.getColor(..)` instead. It does the same thing.

File: app-rating/src/main/java/com/stepstone/apprating/ratingbar/CustomRatingBar.kt [hunk 2]
```diff
@@ -104,6 +112,17 @@ class CustomRatingBar(context: Context, attrs: AttributeSet) : LinearLayout(cont
         this.onRatingBarChangedListener = onRatingBarChangedListener
     }
 
+    private fun getStarColor(context: Context): Int {
+        if (starColorResId != 0) {
+            return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```
Comment (zawadz88) [inline app-rating/src/main/java/com/stepstone/apprating/ratingbar/CustomRatingBar.kt]: Same as above, you could use `ResourcesCompat.getColor()` instead

File: app-rating/src/main/java/com/stepstone/apprating/AppRatingDialogView.kt (hunk 1 above)
Comment (pglebocki) [reply app-rating/src/main/java/com/stepstone/apprating/AppRatingDialogView.kt]: right :)
corrected

File: app-rating/src/main/java/com/stepstone/apprating/ratingbar/CustomRatingBar.kt (hunk 2 above)
Comment (pglebocki) [reply app-rating/src/main/java/com/stepstone/apprating/ratingbar/CustomRatingBar.kt]: corrected

**Comment by pglebocki** [comment]
@R-Sai, I just updated readme docs. I haven't updated visual samples yet, cause there is another feature to be implemented witch required UI changes. I will update UI samples after all implementation.

**Pull request merged**